src/components/Routes/Routes.component.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import HomePage from '../../pages/Home/Home.page';
+import FavoritePage from '../../pages/Favorites/Favorite.page';
+import ErrorPage from '../../pages/Error/Error.page';
+import VideoPage from '../../pages/Video/Video.page';
+import VideoListPage from '../../pages/VideoList/VideoList.page';
+
+export default function Routes() {
+    return (
+        <>
```

Comment:
A Fragment `<></>` is not necessary here since `Switch` is wrapping the whole block of code/nodes.